packages/odyssey-storybook/src/components/odyssey-mui/Tabs/Tabs.stories.tsx

```diff
@@ -75,6 +82,16 @@ const storybookMeta: Meta<TabsProps & TabItemProps> = {
         },
       },
     },
+    onChange: {
+      control: null,
+      description: "Callback fired when the active tab is changed",
+      table: {
+        type: {
+          summary: "func",
+        },
+        defaultValue: "",
```

Comment:
The default value isn't `""`, it's `undefined` right? Do we need to include that or can we remove it?